tests/data_context/cloud_data_context/test_expectation_suite_crud.py

```diff
@@ -625,31 +596,6 @@ def test_expectation_suite_gx_cloud_identifier_requires_id_or_resource_name(
         context.expectations_store._validate_key(key=key)
 
 
-@pytest.mark.cloud
-def test_add_or_update_expectation_suite_updates_existing_obj(
```

Comment:
Removing because `ExpectationsStore` now uses the returned expectation suite to obtain IDs, these tests don't correctly mock it, and this API will be removed soon.